src/Illuminate/Container/BoundMethod.php

```diff
@@ -106,11 +105,12 @@ protected static function normalizeMethod($callback)
      *
      * @param  \Illuminate\Container\Container  $container
      * @param  callable|string  $callback
-     * @param  array $inputData
+     * @param  array  $inputData
      * @return array
+     *
```

Comment:
@GrahamCampbell we don't add these white lines above `@throws` anymore: https://laravel.com/docs/5.7/contributions#coding-style

Reply:
@driesvints Is there any convention for avoiding php 7 return types?

Reply:
@imanghafoori1 The PHP 7 type-hinting is being discussed here: https://github.com/laravel/ideas/issues/1409

Reply:
@driesvints Oh, that's odd, because every throws annotation in the entire codebase has a line above it (bar 1 exception that I noticed, looking through the search results). Are the docs wrong?

<img width="1047" alt="image" src="https://user-images.githubusercontent.com/2829600/50356664-bee66600-054a-11e9-9df9-51c2ab85446d.png">